INSTALLATION.md

```diff
@@ -44,12 +44,14 @@ In order to use the `hubble` command to observe flows you will want to use port-
 cilium hubble port-forward & 
 ```
 
-[Install the Hubble CLI](https://docs.cilium.io/en/v1.11/gettingstarted/hubble_setup/#install-the-hubble-client), and then you can observe flows in the cluster with commands such as: 
+[Install the Hubble CLI](https://docs.cilium.io/en/v1.11/gettingstarted/hubble_setup/#install-the-hubble-client), and then you can continuously observe flows in the cluster with commands such as: 
 
 ```
-hubble observe --server localhost:4245
+hubble observe -f
```

Comment:
I think if you're using port-forward, you have to specify the server address (unless you're on the same machine in which case the default setting of the hubble socket works, but in that case I don't think the hubble port-forwarding would be necessary?)

Reply:
Port forward forwards a remote (or VM/container) port into a localhost port, and as long as the local port is the hubble default (4245) you do not need to specify the server when using port-forward.

Reply:
Aha, the [default server changed ](https://github.com/cilium/hubble/commit/a1957ac11b09dd3bfd525ab3f8300df5623d3352)in 0.9.0 and I still had 0.8.0. You are correct!